lessons/Data-Science/linear-regression/langs/en-us/index.adoc

```diff
@@ -242,7 +242,7 @@ To perform linear regression in CODAP, simply create a scatter plot and select L
 
 @strategy{Going Deeper}{
 
-@ifproglang{pyret}{Students may notice another value in the lr-plot, called @math{R^2}.}@ifproglang{codap}{CODAP displays @math{R^2} rather than @math{r}.} @math{R^2} describes the _percentage of the variation in the y-variable that is explained by least-squares regression on the x variable_. In other words, an @math{R^2} value of 0.20 could mean that “20% of the variation in adoption time is explained by regressing adoption time on the age of the animal”. Discussion of @math{R^2} may be appropriate for older students, or in an AP Statistics class.
+@ifproglang{pyret}{Students may notice another value in the lr-plot, called @math{R^2}.}@ifproglang{codap}{CODAP shows @math{R^2} rather than @math{r}.} @math{R^2} describes the _percentage of the variation in the y-variable that is explained by least-squares regression on the x variable_. In other words, an @math{R^2} value of 0.20 could mean that “20% of the variation in adoption time is explained by regressing adoption time on the age of the animal”. Discussion of @math{R^2} may be appropriate for older students, or in an AP Statistics class.
```

Comment:
```suggestion
@ifproglang{pyret}{Students may notice another value in the `lr-plot`, called @math{R^2}.}@ifproglang{codap}{CODAP shows @math{R^2} rather than @math{r}.} @math{R^2} describes the _percentage of the variation in the y-variable that is explained by least-squares regression on the x variable_. In other words, an @math{R^2} value of 0.20 could mean that “20% of the variation in adoption time is explained by regressing adoption time on the age of the animal”. Discussion of @math{R^2} may be appropriate for older students, or in an AP Statistics class.
```